scripts/ci_policy_checks.rb

```
@@ -38,6 +38,125 @@ def changed_files(base)
+      idx += 1
+      idx += 1 while idx < content.length && content[idx] =~ /[A-Za-z0-9_'.]/
+      tokens << content[start_idx...idx]
+      next
+    end
+
+    tokens << content[idx]
+    idx += 1
+  end
+
+  tokens.join("\n")
+end
```

RuboCop (1.86.0)

[convention] Assignment Branch Condition size for `cpp_token_stream` is too high. [<19, 35, 30> 49.86/23]

(Metrics/AbcSize)